src/main/scala/mesosphere/marathon/core/flow/impl/ReviveOffersActor.scala

```diff
@@ -85,6 +85,11 @@ private[impl] class ReviveOffersActor(
     }
   }
 
+  private[this] def suppressOffers(): Unit = {
+    log.info("=> Suppress offers NOW")
```

Comment:
Log statements are in line with the other ones. 
I will create a ticket which goes over all log lines to remove arrows.